util/topgen/templates/toplevel_memory.ld.tpl

```diff
@@ -17,6 +18,21 @@ def memory_to_flags(memory):
         flags_str += "x"
 
     return flags_str
+
+def flags(mem):
+    swaccess = mem["swaccess"]
+    exec = mem["exec"]
+    assert swaccess in ["ro", "rw"]
```

Comment:
The code in validate also allows `wo`. Is there some extra error checking somewhere else to ensure that a config file can't cause us to hit this assertion?

Reply:
yeah let's remove `wo` from validate for now. I do not really see a reason why a linkable memory region should be `wo`, WDYT?